sbysrc/sby_engine_btor.py

```diff
@@ -105,11 +105,19 @@ def exit_callback(retcode):
                             has_arrays = True
                             break
 
+                setupcmd = "cd %s %s" % (job.workdir, job.cmd_sep)
                 if has_arrays:
-                    setupcmd = "cd %s;" % (job.workdir)
                     finalwit = "engine_%d/trace.wit" % engine_idx
                 else:
-                    setupcmd = "cd %s; { echo sat; btorsim --states model/design_btor.btor engine_%d/trace.wit; } > engine_%d/simtrace.wit &&" % (job.workdir, engine_idx, engine_idx)
+                    if os.name == "posix":
+                        setupcmd = setupcmd + "{ echo sat; btorsim --states model/design_btor.btor engine_%d/trace.wit; } > engine_%d/simtrace.wit &&" %
+                            (engine_idx, engine_idx)
+                    else:
+                        # Windows has no concept of grouping commands into one stream.
+                        # Create and append to the final witness in sequence instead.
```

Comment:
According to https://en.wikibooks.org/wiki/Windows_Batch_Scripting#Redirection its `( ... ) > file` on windows.